### ISSUE TITLE
Currently defined env variables not used - on purpose?

### ISSUE THREAD

**Issue by vovimayhem** [lxs24sxl/vite-plugin-html-env#7]
Hi!

I just noticed that environment variables already defined in the shell (i.e. not dotnet files) are ignored when running `vite build`

Is this on purpose?

**Comment by lxs24sxl** [comment]
Yes, I ignored the environment variables defined in the shell, I thought it would be better to declare the full environment variables in a file beforehand.

**Comment by jmwohl** [comment]
Hmm... unless I'm missing something, this makes it not so useful in many (most?) production situations where env vars are set through the environment, not from a .env file? Maybe I'm misunderstanding though.

**Comment by SphinxKnight** [comment]
seconding @jmwohl here

**Comment by lxs24sxl** [comment]
Version 1.2.8 already supports system environment variables.  @vovimayhem @jmwohl @SphinxKnight
